test/lib/runtime/broker_test.exs

```diff
@@ -450,13 +451,13 @@ defmodule Ask.BrokerTest do
     assert respondent.stats.attempts["mobileweb"] == 1
     assert respondent.state == "active"
 
-    respondent = Repo.get!(Respondent, respondent.id)
```

Comment:
This line was "refreshing" the respondent data. Are you sure skipping that is intended?

Reply:
Yes, I'm sure. We don't need to do this again: https://github.com/instedd/surveda/pull/1585/files#diff-5de71ff0aa29249e2f49bf22956ce6a9R451